values.yaml

```diff
@@ -168,6 +168,12 @@ gateway:
     port: 443
     externalTrafficPolicy: Local
     annotations: {}
+  api:
+    containerPort: 9696
```

Comment:
This should default to the default for gateway, which is that it listens on the same port that gateway does.

Reply:
From what I've experienced, for the headless gateway api the default is 9696. The gateway port that serves traffic through Tyk is 8080.

https://github.com/TykTechnologies/tyk-helm-chart/blob/master/tyk-headless/configs/tyk_mgmt.conf#L3

Are you saying that 9696 is not the default for the gateway api? but something that's manually overridden for the headless configuration?

Reply:
Interesting.. carry on. This doesn't seem to be the tyk default, but its what was done in the chart I guess.